src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,22 @@
+import "../Components-CSS/ArticleCard.css";
+import defaultImg from "../assets/default-pic.jpg"
+
+function ArticleCard({ articles }) {
+    
+  return (
```

Comment:
This is more likely to be an ArticleList rather than ArticleCard. The `<li>` that the map returns sounds more like an ArticleCard